ComponentKit/Core/ComponentUtilities.h

```diff
@@ -75,6 +75,19 @@ namespace CK {
 
     return newVector;
   }
+
+  template <typename T, typename Func>
+  auto intersperse(std::vector<T> a, Func &&func) -> std::vector<T>
```

Comment:
Make this take a "predicate function" that determines if elements from a should actually be added to the result.

this would be used for instance to exclude nil CKStackLayoutItems from the result vector.